python/test.py

```diff
@@ -0,0 +1,6 @@
+
+def addres(addr):
+    print('address: %x', addr)
+
+if __name__ == '__main__':
+    addres(1)
```

Comment:
It may be misspelled. perhaps 'address'?
```suggestion
+    address(1)
```